### PR TITLE
Set latest Ember Data commit version on the app blueprint

### LINKED ISSUE #73: Upgrade ember-data package on octane app blueprint
The Ember Data `master` branch has added improvements for Octane apps:

- `jquery` removal
-  new ED blueprints to use Native syntax

The question is which would be the strategy to set the `ember-data` version on the app blueprint:

1) We could ask the ED team to make a beta release and use it.
2) We could follow the same procedure with `ember-data` as with the `ember-cli/ember.js` packages and pick dynamically the latest commit version from `ember-data/master`.

cc @rwjblue

### PR DESCRIPTION
**Pull request by ppcano**
Closes #73

### CODE REVIEW

**Comment by ppcano** [comment]
cc @rwjblue

File: packages/octane-blueprint-utils/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function getEmberCLIVersionFromTarball() {
     });
 
     got
-      .stream('http://github.com/ember-cli/ember-cli/tarball/master')
```
Comment (NullVoxPopuli) [inline packages/octane-blueprint-utils/index.js]: do all repositories have tarballs?

Comment (NullVoxPopuli) [reply packages/octane-blueprint-utils/index.js]: is this what you get when you click "Download" instead of clone?

Comment (ppcano) [reply packages/octane-blueprint-utils/index.js]: Honestly, no idea; Robert implemented it but I tested and it looks to be working.

Comment (NullVoxPopuli) [reply packages/octane-blueprint-utils/index.js]: I think you need to breach > 50/reqs a minute to see if it works?

Comment (ppcano) [reply packages/octane-blueprint-utils/index.js]: @NullVoxPopuli  Didn't you know that I worked for a Load testing company ;). 

I meant I tested the `getRepoVersion` function. As you pointed, `getEmberCLIVersionFromTarball ` was likely not executed.

**Comment by ppcano** [comment]
CI is failing. I had to `yarn link octane-blueprint-utils` to make it work locally. 

How do you usually work with local packages?

File: packages/octane-blueprint-utils/index.js (hunk 1 above)
Comment (rwjblue) [reply packages/octane-blueprint-utils/index.js]: @NullVoxPopuli tarball download is not throttled

**Pull request merged**